ddl/failtest/fail_db_test.go

```diff
@@ -84,7 +84,8 @@ func (s *testFailDBSuite) SetUpSuite(c *C) {
 }
 
 func (s *testFailDBSuite) TearDownSuite(c *C) {
-	s.se.Execute(context.Background(), "drop database if exists test_db_state")
+	_, err := s.se.Execute(context.Background(), "drop database if exists test_db_state")
```

Comment:
ditto.

Reply:
Since this is in `TearDownSuite`, I think it is ok.